packages/modals/demo/tooltipModal.stories.mdx

```diff
@@ -47,7 +48,16 @@ import { TOOLTIP_MODAL_BODY as BODY } from './stories/data';
       body: { name: 'children', table: { category: 'TooltipModal.Body' } },
       title: { name: 'children', table: { category: 'TooltipModal.Title' } },
       tag: { control: 'text', table: { category: 'TooltipModal.Title' } },
-      'aria-label': { table: { category: 'TooltipModal.Close' } }
+      closeAriaLabel: {
+        name: 'aria-label',
+        type: { required: false, name: 'string' },
+        table: { category: 'TooltipModal.Close' }
+      },
+      titleAriaLabel: {
+        name: 'aria-label',
+        type: { required: false, name: 'string' },
+        table: { category: 'TooltipModal.Title' }
```

Comment:
This control does not belong to the `TooltipModal.Title` component. Rather, it is part of the default set of controls (via prop spread on the story). The `table` can be removed here.

Reply:
Artifact from the first implementation - will remove. :)